### PR TITLE
Updated code snippet

### LINKED ISSUE #38: [Book Feedback]
**Please provide the title of the book**
Blazor for ASP.NET Web Forms

**What do you think could be improved in this book?**
In the example for `Interact with web services` the async keyword is used when it should be await.

```csharp

protected override async Task OnInitializedAsync()
    {
        var client = factory.CreateClient("github");
        var response = await client.GetAsync("repos/dotnet/docs/issues");
        response.EnsureStatusCode();
        var content = async response.Content.ReadAsStringAsync();
    }
```

**What do you like about this book?**
NA.

**Additional context**

### PR DESCRIPTION
**Pull request by sughosneo**
## Summary

Fixes https://github.com/dotnet-architecture/eBooks/issues/38

### CODE REVIEW

File: docs/architecture/blazor-for-web-forms-developers/data.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Data Sources were controls that you could place on a Web Forms page and configur
 
 ![Data Sources](media/data/datasources.png)
 
-ADO.NET is the low-level approach to interacting with a database. Your apps could create a connection to the database with Commands, Recordsets, and Datasets for interacting. The results could then be bound to fields on screen without much code. The drawback of this approach was that each set of ADO.NET objects (`Connection`, `Command`, and `Recordset`) was bound to libraries provided by a database vendor. Use of these components made the code rigid and difficult to migrate to a different database.
+ADO.NET is the low-level approach to interacting with a database. Your apps could create a connection to the database with Commands, Recordsets, and Datasets for interacting. The results could then be bound to fields on screen without much code. The drawback of this approach was that each set of ADO.NET objects (`Connection`, `Command`, and `Recordset`) was bound to libraries provided by a database vendor. Use of these components, made the code rigid and difficult to migrate to a different database.
```
Comment (Youssef1313) [inline docs/architecture/blazor-for-web-forms-developers/data.md:22]: I think the added comma is extra here.

Comment (sughosneo) [reply docs/architecture/blazor-for-web-forms-developers/data.md:22]: Hey @Youssef1313 , wanted to represent "Use of these components" as a single phrase in that sentence and that's the reason thought of adding that comma.

Comment (Youssef1313) [reply docs/architecture/blazor-for-web-forms-developers/data.md:22]: @sughosneo I'm not sure (probably I'm wrong). I'll defer to @gewarren on whether there should be a comma or not.

Comment (nishanil) [reply docs/architecture/blazor-for-web-forms-developers/data.md:22]: drop the comma..

File: docs/architecture/blazor-for-web-forms-developers/data.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ services.AddHttpClient("github", client =>
 });
 ```
 
-Whenever you need to access data from GitHub, create a client with a name of `github`. The client is configured with the base address, and the request headers are set appropriately. Inject the `IHttpClientFactory` into your Blazor components with the `@inject` directive or an `[Inject]` attribute on a property. Create your named client and interact with services using the following syntax:
+Whenever you need to access data from GitHub, create a client with the name of `github`. The client is configured with the base address, and the request headers are set appropriately. Inject the `IHttpClientFactory` into your Blazor components with the `@inject` directive or an `[Inject]` attribute on a property. Create your named client and interact with services using the following syntax:
```
Comment (nishanil) [inline docs/architecture/blazor-for-web-forms-developers/data.md:112]: I think, `a` is fine.

File: docs/architecture/blazor-for-web-forms-developers/data.md (hunk 1 above)
Comment (sughosneo) [reply docs/architecture/blazor-for-web-forms-developers/data.md]: Thanks @Youssef1313 and @nishanil.  Done

File: docs/architecture/blazor-for-web-forms-developers/data.md (hunk 2 above)
Comment (sughosneo) [reply docs/architecture/blazor-for-web-forms-developers/data.md]: Done.

**Review by nishanil — APPROVED** [review]

**Pull request merged**